draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1065,76 +1065,152 @@ SDP Answer
-          RTCP packet can contain several SSRC to identification-tag mappings. The offerer
-          and answerer maintain tables mapping RTP streams identified by SSRC,
-          to “m=“ lines identified by the identification-tag. These tables are
-          updated each time an RTP/RTCP packet containing one or more mappings
-          from SSRC to identification-tag is received. Note that the mapping
-          from SSRC to identification-tag can change at any time during an RTP
-          session.
+  
+      <t> As described in <xref target="RFC3550"/>, RTP/RTCP packets
+      are associated with RTP streams <xref target="RFC7656"/>. Each
+      RTP stream is identified by an SSRC value, and each RTP/RTCP
+      packet carries an SSRC value that is used to associate the
+      packet with the correct RTP stream (an RTCP packet can carry
+      multiple SSRC values, and might therefore be associated with
+      multiple RTP streams).</t>
+
+      <t> In order to be able to process received RTP/RTCP packets
+      correctly it must be possible to associate an RTP stream with
+      the correct "m=" line, as the "m=" line and SDP attributes
+      associated with the "m=" line contain information needed to
+      process the packets.</t>
+
+      <t> As all RTP streams associated with a BUNDLE group are using
+      the same address:port combination for sending and receiving
+      RTP/RTCP packets, the local address:port combination cannot be
+      used to associate an RTP stream with the correct "m=" line. In
+      addition, multiple RTP streams might be associated with the same
+      "m=" line.</t>
+
+      <t> Also, as described in <xref target="sec-rtp-sessions-pt"/>,
+      the same payload type value might be used by multiple RTP
+      streams, in which case the payload type value cannot be used to
+      associate an RTP stream with the correct "m=" line. </t>
+ 
+      <t> In order for an offerer and answerer to always be able to
+      associate an RTP stream with the correct "m=" line, the offerer
+      and answerer using the BUNDLE extension MUST support the
+      mechanism defined in <xref format="default" pageno="false"
+      target="sec-receiver-id"/>, where the offerer and answerer
+      insert the identification-tag (provided by the remote peer)
+      associated with an "m=" line in RTP and RTCP packets associated
+      with a BUNDLE group. </t>
+  
+      <t>When an RTP packet is received by a transport, that packet
+      needs to be routed to the correct RtpReceiver. The following
+      steps MUST be followed to prepare to route packets:</t>
+      
+      <t>
+        <list>
+
+          <t>Construct a table mapping MID to "m=" line for each "m="
+          line configured to receive from this transport.</t>
+
+          <t>Construct a table mapping incoming SSRC to "m=" line for
+          each "m=" line configured to receive from this transport and

```

Comment:
This said RtpReceiver before, right? I do agree you need to change this but m= line's aren't configure to receive from transport. I believe you need to use "associated with" or something, in this draft.